perceval/backends/core/hyperkitty.py

```diff
@@ -79,10 +81,11 @@ def fetch(self, category=CATEGORY_MESSAGE, from_date=DEFAULT_DATETIME):
 
         :param category: the category of items to fetch
         :param from_date: obtain messages since this date
+        :param to_date: obtain messages until this date
 
         :returns: a generator of messages
         """
```

Comment:
Missing condition to check if `to_date` is None. That condition is another backends like git:
```
if not to_date:
    to_date = DEFAULT_LAST_DATETIME
```